pkg/api/core/v1/client/apps.go

```diff
@@ -27,6 +27,7 @@ import (
 	api "github.com/epinio/epinio/internal/api/v1"
 	"github.com/epinio/epinio/internal/duration"
 	"github.com/epinio/epinio/pkg/api/core/v1/models"
+	progressbar "github.com/schollz/progressbar/v3"
```

Comment:
The existing internal `helpers/termui` has progress functionality.

Reply:
Ah, cool, I didn't know. I'll have a look.

Reply:
Oh, I see. I had not realized that we already had this package for `client-sync` too.
With that, agreed to keeping as-is.